pkg/database/openshiftclusters.go

```diff
@@ -289,6 +290,19 @@ func (c *openShiftClusters) EndLease(ctx context.Context, key string, provisioni
 	}, nil)
 }
 
+func (c *openShiftClusters) QueryAll(ctx context.Context, partitionKey string) (*api.OpenShiftClusterDocuments, error) {
+	docs, err := c.c.QueryAll(ctx, partitionKey, &cosmosdb.Query{
+		Query: `SELECT * FROM OpenShiftClusters doc`,
+	}, nil)
+	if err != nil {
+		return nil, err
+	}
+	if docs == nil {
```

Comment:
If `docs == nil`, you don't need a special return case; two nils will be returned at line 303 anyway: `return docs, nil`

Reply:
Thanks !